### ISSUE TITLE
display the new flyctl version number after `fly version update`

### ISSUE THREAD

**Issue by catflydotio** [superfly/flyctl#2017]
flyctl doesn't indicate which version we've just updated to:

```
$ fly version update
Update available 0.0.504-pre-3 -> v0.0.505.
Run "fly version update" to upgrade.
/bin/zsh -c
Running automatic update [curl -L "https://fly.io/install.sh" | sh]
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1475    0  1475    0     0   3635      0 --:--:-- --:--:-- --:--:--  3660
######################################################################## 100.0%
set channel to shell
flyctl was installed successfully to /Users/chris/.fly/bin/flyctl
Run 'flyctl --help' to get started
```

There's already issue #1444 about the redundant "update available" message.

I always run `fly version` after an update to see what I ended up getting:

```
$ fly version
fly v0.0.506 darwin/arm64 Commit: 137323a9 BuildDate: 2023-04-05T08:38:51Z
```

**Comment by catflydotio** [comment]
This must have been reverted somewhere but I don't know if it was because there wasn't a great solution or because it just got lost somehow.

**Comment by alichay** [comment]
yeah. the code technically exists but it always fails and the error message is just hidden 😬
if anyone wants to take over feel free, but be warned, getting flyctl to reliably grab the new version number might be a trickier problem to solve than building a cloud paas

**Comment by catflydotio** [comment]
Could flyctl check GitHub for the version number under the `latest` tag, and then declare its intention to update to that version?

**Comment by catflydotio** [comment]
Anyway, that was just my maybe-don't-close-this-issue-yet argument. I'm not worried about us leaving this one alone for now.

**Comment by tomr-a5** [comment]
[#2032](https://github.com/superfly/flyctl/pull/2032) added the post-upgrade version output, and [#3441](https://github.com/superfly/flyctl/pull/3441) fixed the lookup that kept it from printing reliably. I’m going to close this as fixed. Please reopen if the latest flyctl still does not show the upgraded version.
